### DISCUSSION TITLE
New Module to be Released for Beta soon, Multilingual Text Module

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#4321] [Show and tell]
New module **MLTextHTML** that will be released as beta for the first few weeks. This is a multilingual Text / HTML module that allows you to add multilingual content to your site.  Just add a language, using the language manager and that is it, you are ready to add content. 

![Screenshot 2024-06-10 122034](https://github.com/oqtane/oqtane.framework/assets/9447612/87af0e56-771d-4af7-824c-50dd1f079306)

![Screenshot 2024-06-10 122143](https://github.com/oqtane/oqtane.framework/assets/9447612/d971fb15-ecf9-42a9-b949-ea33e14697cf)

![Screenshot 2024-06-10 122627](https://github.com/oqtane/oqtane.framework/assets/9447612/aaa02e80-6237-4a7f-987e-f3ab5afe425f)

![MLTextHTML](https://github.com/oqtane/oqtane.framework/assets/9447612/9b9fb1bd-3921-4ced-8012-62ddf126d71b)

**Comment by sbwalker** [comment]
@leigh-pointer just a note on this... there is no plan for Oqtane to support content localization within a site (ie. the approach taken by DNN). This topic has been covered previously in a variety of public issues/discussions. If content localization is implemented in the future it will be based on the multi-site (ie. multi-tenant) capability (ie. each site has the ability to localize every element of content). I am mentioning this because although modules do have the ability to support content localization (as you have done in the MLHtmlTextmodule), other framework services such as Site/Page attributes, Search, etc... will not support this approach. This may not be a problem depending on your specific implementation... but I thought I would mention it for future consideration.

**Reply by leigh-pointer** [reply]
@sbwalker Thank you for the clarification regarding Oqtane's approach to content localization. It's helpful to understand the current framework's stance on this matter. I'll keep this in mind as I develop my new module and consider the implications for its future integration with Oqtane's capabilities. While my module does support content localization, I'll ensure that its functionality aligns with Oqtane's multi-site capability for localization if such integration becomes necessary in the future. Appreciate the heads up!
